.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        # 11 runtime needed for Sonar, we still build with 8 as a target
+        java-version: 11
+    - name: Grant execute permission for gradlew
```

Comment:
that's weird, issn't it part of git attributes?

Reply:
I'll try without, I took the template generated with GH itself